resources/views/news/edit.blade.php

```diff
@@ -1,31 +1,55 @@
 @extends('layouts.app')
 
 @section('content')
-    <div>
+  <div class="container md:w-[800px] mx-auto">
+    @auth
+      <form action="{{ route('news.update', $news) }}" method="post" enctype="multipart/form-data"
+        class="bg-white p-6 mt-20 rounded-20 shadow-xl" id="editNews">
+        @method('PUT')
+        @csrf
         <div>
-            <h1>Edit News</h1>
-            @auth
-                <form method="post" action="{{ route('news.update', $news) }}" id="editNews">
-                    @method('PUT')
-                    @csrf
-                    <div>
-                        <label for="name">Title:</label>
-                        <input type="text" id="title" name="title" value="{{ old('title', $news->title) }}">
-                        <x-input-error for="name"/>
-                    </div>
-                    <label for="editor">Content:</label>
-                    <div class="flex flex-col space-y-2">
-                        <div id="editor" class="mt-1 block w-full rounded-md border-gray-300 shadow-sm"></div>
-                    </div>
-                    <x-input-error for="editor"/>
-                    <input type="hidden" id="oldText" value="{{ $news->text }}">
-                    <input type="hidden" name="text" id="text">
-                    <div>
-                        <button type="submit">Update</button>
-                    </div>
-                </form>
-            @endauth
+          <div>
+            <h3 class="text-xl leading-6 font-medium text-gray-900">
+              Edit News
+            </h3>
+          </div>
+          <div class="mt-6 flex flex-col gap-7">
+            <div x-data class="sm:flex items-center">
+              <img x-ref="image" id="image" src="{{ asset('/static/images/no_image.webp') }}" alt="news_avatar"
+                class="w-full sm:w-[400px] h-[200px]">
+              <input @change="image.src = URL.createObjectURL($event.target.files[0])" type="file" accept="image/*"
+                id="logo" name="logo" class="hidden">
+              <label for="logo"
+                class="inline-flex items-center px-4 py-2 border border-transparent shadow-sm text-base font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 cursor-pointer mx-auto w-full justify-center mt-3 sm:w-auto sm:mt-0">
+                <i class="fa-solid fa-arrow-up-from-bracket mr-3"></i>
```

Comment:
Can we show existing image?

Reply:
My teammate @MichalMyskow has just fixed that